src/pages/AdminCircleApiPage/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { circleApiPermissionsSelector } from './useCircleApiKeys';
+
+export const API_PERMISSION_LABELS: Record<
```

Comment:
Can you help me understand what's going on here? What's the benefit of this type? There are other string literals (eg the explainer text) that isn't folded into a type, so I'm trying to understand what you were going for here? It seems nice but I feel like i'm missing something.

Reply:
I guess this is mapping the keys of the selector to friendly strings?

Reply:
The purpose of typing it with a `Record` is to ensure that every possible value from circleApiPermissions has a corresponding friendly string. If I were to remove one of those lines, it would give a type error like this: 
![image](https://user-images.githubusercontent.com/7143583/176033440-1d8c5a98-cf6c-482a-92af-9a792426554e.png)


This makes sure that if we add any new permissions, we also add in a friendly string for it.